test/WebJobs.Script.Tests.Integration/TestScripts/Node/HttpTriggerReturn/index.js

```diff
@@ -0,0 +1,3 @@
+﻿module.exports = function (context, req) {
+    context.res.type("text/plain").send("test");
```

Comment:
I'm not sure this actually makes sense in the context of a $return binding. $return means the response is set to the value returned via context.done(null, x). To make this work with your fluent APIs, I'd expect a way for users to create a response object via your APIs and returning via context.done(null, res).